### PR TITLE
add build mode info to build guide

### PR DESCRIPTION
**Pull request by xeTaiz**
According to discussions [in this issue on the main repo](https://github.com/inviwo/inviwo/issues/882)

### CODE REVIEW

File: pages/manual/manual-gettingstarted-build.md [hunk 1]
```diff
@@ -29,6 +29,13 @@ The `--recurse-submodules` is necessary to pull dependencies.
  Use front slashes and no space between modules. Configure again. External modules are developed in the [inviwo modules repository](https://github.com/inviwo/modules).
 4. Hit Generate and open the project in your IDE.
 
+{% include note.html content="
+When using a multi-configuration generator (like Visual Studio and most IDEs) you may want to adjust your build mode manually, since it probably defaults to `Debug` at first, which has a large impact on the performance.
+If you use a single-configuration generator, you can control the build mode using `CMAKE_BUILD_TYPE` in CMake, which defaults to `Release`
```
Comment (ResearchDaniel) [inline pages/manual/manual-gettingstarted-build.md]: I'm not sure if it defaults to release. We could just link to:
https://cmake.org/cmake/help/latest/variable/CMAKE_BUILD_TYPE.html

**Comment by xeTaiz** [comment]
[click for properly rendered screenshot](https://imgur.com/a/xoFb7Uz)
(See "Note" boxes and last section)

File: pages/manual/manual-gettingstarted-build.md [hunk 2]
```diff
@@ -29,6 +29,13 @@ The `--recurse-submodules` is necessary to pull dependencies.
  Use front slashes and no space between modules. Configure again. External modules are developed in the [inviwo modules repository](https://github.com/inviwo/modules).
 4. Hit Generate and open the project in your IDE.
 
+{% include note.html content="
+When using a multi-configuration generator (like Visual Studio and most IDEs) you may want to adjust your build mode manually, since it probably defaults to `Debug` at first, which has a large impact on the performance.
```
Comment (ResearchDaniel) [inline pages/manual/manual-gettingstarted-build.md]: **Recommendation: Set build type to RelWithDebInfo** 
Explanation goes here

Link to how to do it in Visual Studio: https://docs.microsoft.com/en-us/visualstudio/debugger/how-to-set-debug-and-release-configurations?view=vs-2019

**Review by ResearchDaniel — APPROVED** [review]
Thanks, I had some small comments

File: pages/manual/manual-gettingstarted-build.md (hunk 1 above)
Comment (xeTaiz) [reply pages/manual/manual-gettingstarted-build.md]: I have looked into it a bit. the default is empty and apparently empty defaults to Release (according to some stack overflow pages.. could not find any official information on this")

Comment (xeTaiz) [reply pages/manual/manual-gettingstarted-build.md]: more investigation actually gave some more insight. Leaving this empty defaults to build mode without optimization and without debug symbols. We should just set a default to this property for all single-generator users... I'll adjust this accordingly..

**Pull request merged**